### DISCUSSION TITLE
230912 스프린트 회의 (14:00 ~ )

### DISCUSSION THREAD

**Discussion by somsom13** [woowacourse-teams/2023-shook#402] [회의]
## 스프린트 회의 
이번 스프린트 기간 동안 해야 할 작업을 결정한다.

**Comment by Cyma-s** [comment]
## 회위회 - 스프린트 목적을 정하는 회의를 위한 회의

- 예상 목적
    - 스프린트 목적을 정한다. (토의 후 의사 결정)
- 예상 결과
    - 스프린트 목적이 정해진다.
- 예상 시간
    - 20분

## 스프린트 목적을 정하는 회의

2023년 9월 12일 오후 2:07 

### 데모데이 요구사항

**웹 백엔드**

- 톰캣 설정 중 아래 값을 적절하게 설정하고, 해당 값으로 설정한 이유를 공유
    - threads max
    - max connections
    - accept count
- 서비스에서 사용하는 쿼리를 정리하고, 각 쿼리에서 사용하는 인덱스 설정
    - 서비스에서 사용하는 모든 조회 쿼리와 테이블에 설정한 인덱스 공유
    - 인덱스를 설정할 수 없는 쿼리가 있는 경우, 인덱스를 설정할 수 없는 이유 공유

**웹 프론트엔드**

- 목표했던 타겟 환경에서의 서비스 핵심 기능 동작 확인
- 성능 리포트 작성 및 리포트 기준으로 필요한 만큼의 성능 최적화 작업 진행
    - 구체적인 성능 개선 대상과 선정한 이유, 목표 지표를 설정한 이유를 반드시 포함한다.

**체크리스트 만들기**

https://woowacourse.slack.com/archives/C04KDGGRNQ1/p1694158432170419

5차 데모데이 체크리스트

### 스프린트 목적

- 5차 데모데이 요구사항을 모두 충족한다. ⇒ 요구사항을 충족하면서 성능 최적화가 될 것.
    - 협업
        - 로그인 리프레시 토큰 처리
        - 카카오 로그인 추가
        - 내부적으로 토큰을 어떻게 다룰 것인지 하나로 정리하자.

**마무리  2023년 9월 12일 오후 2:23**
